packages/workbox-strategies/src/NetworkOnly.ts

```diff
@@ -30,6 +34,28 @@ import './_version.js';
  * @memberof module:workbox-strategies
  */
 class NetworkOnly extends Strategy {
+  private readonly _networkTimeoutSeconds: number;
+
+  /**
+   * @param {Object} options
```

Comment:
Note, all these options are optional, but they're not indicated as such by the jsdocs.

I've made that change as part of my precaching-v6 changes, but you may as well make them here to avoid merge conflicts: https://github.com/GoogleChrome/workbox/compare/v6...precaching-v6#diff-0c320ed20894cc8a17c07ff33737889a

Reply:
Done.